### ISSUE TITLE
Alchemy Specialization Elixir Mastery quest

### ISSUE THREAD

**Issue by cperwin42** [dalaranwow/dalaran-wow#4357]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Trying to do the Elixir Master Alchemy specialization quest. The npc Lorokeem does not have the quest like he should.
**Expected behaviour**: Tell us what should happen instead
I would like to be able to obtain the quest to be come an Elixir Master.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**
http://www.wowhead.com/quest=10902/master-of-elixirs
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
That was from the cata update you need to go to outlands to pick up the quest.

**Comment by TheRealestMcCoy** [comment]
GO talk to http://wowwiki.wikia.com/wiki/Alchemist_Gribble

**Comment by Rushor** [comment]
yes true you get http://wotlk.openwow.com/quest=10906 from http://wotlk.openwow.com/npc=18802 or http://wotlk.openwow.com/npc=16588. but the follow-up quest can only be accepted from http://wotlk.openwow.com/npc=19052
